### ISSUE TITLE
Arch Linux after update to mariadb and qt frontend cutlist editor not working: related to issue #373 below

### ISSUE THREAD

**Issue by glenb77** [MythTV/mythtv#381]
On Arch Linux and most likely only Arch, after recent issue with backend not loading after update to mariadb 10.6.3 and the related fix from arch devs in QT, updating all packages and rebuilding Myth, the backend does in fact run as usual. However when in any frontend and in a playing recording, using 'E' key or selecting edit from the menus, the cutlist editor does not appear instead a message stating 'no seektable found' appears. Rebuilding seektable does nothing. Most important the seektable does actually exist as i am able to move around forward and back in all recordings. In Kodi - MythPvr i can see the flagged cutpoints in the very same shows. This is from the Arch AUR - Mythtv package which is 31-fixes. This issue has been confirmed by one other user of the package. See https://aur.archlinux.org/packages/mythtv/ in the comments area.

To reproduce:
Install Arch Linux with all current updates. 
Build Mythtv with AUR package
Use Frontend. play a recording and go to cutlist editor (hit 'e' on keyboard)

i did not have a log to see what the error was, and shortly after I reverted back and went to version of mariadb that was working and the editor works like before on same recordings. If needed, I can backup, upgrade mariadb, and run frontend in a terminal and report back with some logging.

Thanks.

* **Platform**:

* **MythTV version**:

* **Package version**:

* **Component**:

**Comment by paul-h** [comment]
pkgver=31+fixes.20210401.563a05b7

Looks like you need to contact the package author to pull a newer version that has the fixes for the problem where MariaDB changed 'offset' to be a reserved word.

**Comment by glenb77** [comment]
I doubt it, the way the AUR works is that the package you build is current from the most current 'fixes' branch. the date back in April was the time he changed any local patches or the Arch PKGBUILD itself. i don't have it running anymore, but the build has the version number and it would be most current and I believe the change with the reserved word were pushed to fixes branch.

The package pulls the following git reference:
"git+https://github.com/MythTV/mythtv#branch=fixes/31"

It would have been the most current fixes when I did this 2 days ago. I will do again, or build from git without the Arch Package to confirm, but I am certain of how the AUR package is working and the sources used.

**Comment by paul-h** [comment]
OK I assumed that was the MythTV version the packs was built from. That is a valid git version from the fixes branch.

The fix was committed on July 31 
https://github.com/MythTV/mythtv/commit/5824c588db24b4e71a7d94e829e6419f71089297

**Comment by glenb77** [comment]
As soon as i get some I will rebuild and update again and maybe i could get hint from a terminal run of frontend. The new mariadb also created some issues with nextcloud which i also run. I'll follow up with more if come up with anything. Thanks..

**Comment by glenb77** [comment]
I updated mariadb to 10.6.4 and did the mysql update after. I rebuilt Mythtv and the editor is working. I did get failures on the mysql update the earlier time so maybe that accounted for the problem. It is working correctly now. The android app off the updated backend will not bring up the editor which I would have expected, but the frontends with the rebuilt Myth and the updated mariadb are working. This issue can be closed
